lib/config_cat/supervisor.ex

```diff
@@ -25,8 +24,11 @@ defmodule ConfigCat.Supervisor do
       |> Keyword.merge(options)
       |> generate_cache_key(sdk_key)
 
-    name = Keyword.fetch!(options, :name)
-    Supervisor.start_link(__MODULE__, options, name: name)
+    # Rename name -> instance_id for everything downstream
```

Comment:
I chose to keep the `:name` option that end user's will use so that this wasn't a breaking change. I then rename it to `instance_id` for everything internal as that better communicates its purpose.